rs/replicated_state/src/canister_state/system_state.rs

```diff
@@ -340,6 +343,7 @@ pub struct SystemState {
     canister_history: CanisterHistory,
```

Comment:
I wonder if we should be skipping this, as this could be quite big. Same with `canister_log` below`.

Reply:
Added recursive.